go/controller/internal/httpserver/server.go

```diff
@@ -138,6 +139,9 @@ func (s *HTTPServer) setupRoutes() {
 	s.router.HandleFunc(APIPathTeams+"/{teamID}", adaptHandler(s.handlers.Teams.HandleGetTeam)).Methods(http.MethodGet)
 	s.router.HandleFunc(APIPathTeams+"/{teamLabel}", adaptHandler(s.handlers.Teams.HandleDeleteTeam)).Methods(http.MethodDelete)
 
+	// Agents
+	s.router.HandleFunc(APIPathAgents+"/{agentId}/invoke", adaptHandler(s.handlers.Invoke.HandleInvokeAgent)).Methods(http.MethodPost)
```

Comment:
we should have two paths here, one for `invokeSync` and other one for `invokeAsync` (the backend HandleInvokeAgent can be a single func with an if statement as you have it right now).

Reply:
I wonder if there's prior art for APIs like this. I agree that having separate paths feels cleaner, but the more I think about this I wonder how others have done it, and if that's something we can learn from

Reply:
Maybe something along the lines of `invoke` and `start`, or something like that.